namespace-provisioner/use-case7.hbs.md

```diff
@@ -0,0 +1,471 @@
+setting up Kubernetes secrets to securely pass these credentials to the relevant resources.
+
+This section provides instructions on configuring the `default` service account to
+interact with Git repositories for workloads and supply chain using Namespace Provisioner.
+
+To set up the service account to interact with Git repositories, follow the steps below:
+
+1. Create a secret in the `tap-install` namespace or any preferred namespace, containing Git credentials in YAML format.
+
+   - `host`, `username`, `caFile` and `password` or `personal access token` values for HTTP-based Git Authentication.
+   - `ssh-privatekey`, `identity`, `identity_pub`, and `known_hosts` for SSH-based Git Authentication.
+
+    >**Note:** The `stringData` key of the secret must end with either **.yaml** or **.yml** suffix.
+
+    Using HTTP(s) based Authentication
+    : If using Username and Password for authentication.
+
+
+       Assuming that, in this configuration for an air-gapped environment,
+       the Git Repository server has a custom certificate of authority that
+       cannot be verified against public issuers, you must provide
+       the `caFile` content to log in against it.
+
+
+      ```yaml
+      cat << EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: workload-git-auth
+        namespace: tap-install
+      type: Opaque
+      stringData:
+        content.yaml: |
+          git:
+            #! For HTTP Auth. Recommend using https:// for the git server.
+            host: GIT-SERVER
+            username: GIT-USERNAME
+            password: GIT-PASSWORD
+            caFile: |
+              -----BEGIN CERTIFICATE-----
+              ...
+              -----END CERTIFICATE-----
+      EOF
+      ```
+
+    Using SSH based Authentication
+    : If you prefer using an SSH private key for authentication, create the Git secret with the authentication details as follows:
+
+
+      ```yaml
+      cat << EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: workload-git-auth
+        namespace: tap-install
+      type: Opaque
+      stringData:
+        content.yaml: |
+          git:
+            host: GIT-SERVER
+            #! For SSH Auth
+            ssh_privatekey: SSH-PRIVATE-KEY
+            identity: SSH-PRIVATE-KEY
+            identity_pub: SSH-PUBLIC-KEY
+            known_hosts: GIT-SERVER-PUBLIC-KEYS
+      EOF
+      ```
+
+2. To create a secret to be added to the service account in the developer namespace within the GitOps repository, you can use this [example](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/gitops-airgap/resources/git.yaml) for HTTP-based or this [example](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/gitops-airgap/resources/settings.xml.yaml) for `setings.xml`-based, or follow the provided example below.
```

Comment:
both the links showed 404

Reply:
Links would work when accelerator [PR](https://github.com/vmware-tanzu/application-accelerator-samples/pull/378) is merged